src/hapi/pipelines/database/org_type.py

```diff
@@ -29,6 +29,9 @@ def populate(self):
         logger.info("Populating org type table")
 
         def parse_org_type_values(code: str, description: str) -> None:
+            self.data[code] = code
```

Comment:
Why do you also need unnormalised strings as keys? I guess that was the whole point of this PR aha, I don't see where it will be used though

Reply:
I think the org type name we get from the data is freeform, so it could be that we are able to match it as is (thus saving one normalise call), but if not, we then normalise it and try looking up again (in `get_code_from_name`). If we find it after normalising we add a lookup from the unnormalised name to the mapping to avoid having to normalise again in future if the same org type name shows up again.

Reply:
That's clear thanks